app/src/main/java/com/codesoom/assignment/TaskHttpHandler.java

```diff
@@ -0,0 +1,100 @@
+        String method = exchange.getRequestMethod();
+        URI uri = exchange.getRequestURI();
+        String path = uri.getPath();
+
+        String content = null;
+        String request = parsingRequest(exchange);
+
+        if (method.equals("GET")) {
+            content = tasksToJson();
+            exchange.sendResponseHeaders(200, content.getBytes().length);
+        } else if (method.equals("POST")) {
+            if (!request.isBlank()) {
+                Task task = toTask(request);
+                tasks.add(task);
+                Task storedTask = tasks.stream()
+                        .filter(t -> t.getId().equals(id - 1))
+                        .findFirst()
+                        .orElseThrow(() -> new RuntimeException("생성 과정 중 오류가 발생했습니다."));
+                content = taskToJson(storedTask);
+                exchange.sendResponseHeaders(201, content.getBytes().length);
+            }
+        } else if (method.equals("PUT") && path.matches("/tasks/[0-9]+")) {
+            String[] splitedPath = path.split("/");
+            Long findId = Long.valueOf(splitedPath[2]);
+
+            if (!request.isBlank()) {
+                Task storedTask = tasks.stream()
+                        .filter(t -> t.getId().equals(findId))
+                        .findFirst()
+                        .orElseThrow(() -> new RuntimeException("찾을 수 없는 Task입니다."));
+                storedTask.setTitle("변경되었습니당.");
+                content = taskToJson(storedTask);
+                exchange.sendResponseHeaders(200, content.getBytes().length);
+            } else {
+                content = "올바른 Request 요청이 아닙니다.";
+                exchange.sendResponseHeaders(400, content.getBytes().length);
+            }
+        } else if (method.equals("DELETE")) {
+
+        } else {
+            content = "Bad Request";
+            exchange.sendResponseHeaders(400, content.getBytes().length);
+        }
+
+        OutputStream responseBody = exchange.getResponseBody();
+        responseBody.write(content.getBytes());
+        responseBody.flush();
+        responseBody.close();
+    }
+    // 요청온 requestBody를 파싱해주는 기능
+    private String parsingRequest(HttpExchange exchange) {
+        return new BufferedReader(new InputStreamReader(exchange.getRequestBody()))
+                .lines()
+                .collect(Collectors.joining("\n"));
+    }
+
+    // 요청온 경로가 유효한지 확인해서 boolean 반환
+    private boolean validPath(String path) {
+        return path.equals("/tasks");
+    }
+
+    // 요청 받은 content를 Task 객체로 매핑하고 리턴
+    private Task toTask(String content) throws JsonProcessingException {
+        Task task = objectMapper.readValue(content, Task.class);
+        task.setId(id++);
+        return task;
+    }
+    // 저장되어 있는 tasks을 Json으로 변환하는 메서드
+    private String tasksToJson() throws IOException {
+        return objectMapper.writeValueAsString(tasks);
+    }
+    // 하나의 Task를 Json으로 반환해주는 메서드
```

Comment:
```suggestion
  /**
   * Task를 Json String으로 변환하여 리턴합니다.
   * 
   * @param 
   * @throws
   **/
    // 하나의 Task를 Json으로 반환해주는 메서드
```

javaDoc 명세를 한번 따라보는 것이 좋은 도움이 될 거에요!
위에 `@param` `@throws` 도 한번 채워보아요!

Reply:
JavaDoc으로 작성해봤습니다.
JavaDoc을 사용하면서 스스로 메서드의 역할, 파라미터, 예외, 리턴 값까지 한번 더 생각해볼 수 있어서 좋은 것 같아요.
다른 것들에도 적용해보는 습관을 가져보겠습니다.
[c20b65b](https://github.com/CodeSoom/spring-week1-assignment-1/pull/107/commits/c20b65bb8d6e3b8f9333400971bae250d3bd7e4d)